### DISCUSSION TITLE
Start Pluto from a script using a different browser than the system default

### DISCUSSION THREAD

**Discussion by wsgreen** [fonsp/Pluto.jl#955] [Questions about using Pluto]
I am trying to start Pluto from a shell script:
```
#!/usr/bin/env julia
using Pluto
Pluto.run()
```
However, I would like to start it using a different browser than the default on my system.
Pluto uses xdg-open on Linux. I can change the default browser using xdg-settings in the script but that change would be global.
I can use `Pluto.run(launch_browser=false)` but then how do I get the correct URL to start the browser since `run()` blocks the script?

I guess it would be nice if I could do something like `Pluto.run(browser="/usr/bin/firefox")`, but maybe there is another way to do this.

Thanks!

**Comment by fonsp** [comment]
We decided not to support this feature with the reasoning that your default browser is likely your preferred browser.

**Reply by wsgreen** [reply]
Fair enough, it could be just me :)

I tried to find a workaround, but so far it was harder than I expected.
